lib/_http_agent.js

```diff
@@ -130,6 +130,9 @@ Agent.prototype.getName = function getName(options) {
   if (options.family === 4 || options.family === 6)
     name += ':' + options.family;
 
+  if (options.socketPath)
+    name += ':' + options.socketPath;
```

Comment:
I think it makes sense to extend `test/parallel/test-http-agent-getname.js` and cover this.

Reply:
Nit: the socketPath should not be set when a family, a port or a localAddress is set. Therefore it would be nicer branch wise to write
```js
if (options.socketPath) {
 name += `:::${options.socketPath}`;
} else {
  if (options.port)
    // ...
  if (options.localAddress)
    // ...
  // ...
}
```
I think it is similar with `servername` down below.